dask/distributed/tests/test_distributed.py

```diff
@@ -2,16 +2,20 @@
 
 import time
 from subprocess import Popen, PIPE
+from distutils.version import StrictVersion
```

Comment:
How about a try-except block with `ImportError` instead of depending on distutils here.  There is also this

```
In [2]: IPython.__version__
Out[2]: '4.0.0'
```